### DISCUSSION TITLE
Black border of Image when loading page

### DISCUSSION THREAD

**Discussion by peterburgs** [chakra-ui/chakra-ui#7745] [Q&A]
I'm implementing a feature that has an image background and some text.
Whenever I load the page, the image always has a black border appears and then disappear.
![image](https://github.com/chakra-ui/chakra-ui/assets/48356852/0bc372e1-8398-48ff-9048-7f5843af9ef5)
![image](https://github.com/chakra-ui/chakra-ui/assets/48356852/b948b74a-9e56-45bf-988c-ce7c1ce793c8)

Here is my code:
```
import {Image} from "@chakra-ui/react";

...

Box position={"absolute"} width={"100%"} height={"100%"}>
  <Image
	  src={backgroundSrc}
	  height={"100%"}
	  width={"100%"}
	  objectFit={"cover"}
	  loading={"lazy"}
	  alt={""}
	  outline={"none"}
	  border={"none"}
	  style={{
		  border: "none",
	  }}
  />
</Box>
```

Can anyone help me to remove the ugly black border?

**Comment by 1mlm** [comment]
Same problem here, whenever I add an input (code blow) for example or an accordeon, it has those black borders when loading the page and then it dissappears.

https://github.com/chakra-ui/chakra-ui/assets/71925280/54fc3eb2-a679-47f0-96b5-842d32b2aac6
